Plugin~/WebRTCPlugin/Context.cpp

```diff
@@ -79,7 +79,7 @@ namespace webrtc
 #pragma region open an encode session
     uint32_t Context::s_encoderId = 0;
     uint32_t Context::GenerateUniqueId() { return s_encoderId++; }
-#pragma endregion 
+#pragma endregion
```

Comment:
I'm removing extra whitespace in files I touched, so some no-op changes like this will be evident in the diff.